python/MooseDocs/commands/init.py

```diff
@@ -0,0 +1,125 @@
+#* This file is part of the MOOSE framework
```

Comment:
So, because we have this init utility, we don't need to set up any templates within stork (modules or app)? We would just be directing users to perform an init command for their application and category. Is there doing to be any documentation (perhaps near getting started) for this?

Reply:
For now I am just intending this to be used by me. I have 4 or 5 more apps to setup. Perhaps, if it works for those I will add a bit of documentation regarding this command. Currently the docs for SQA mention contacting the MOOSE team for help, which is fine for now.